src/components/ImageFOL/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+import { NextRouter, useRouter } from 'next/router';
+import Link from 'next/link';
+import {
+    defaultFadeInLeftVariants,
+    defaultFadeInUpVariants,
+} from 'src/constants/motion';
+interface IImageFOL {
+    month: string;
+    quantity: number;
+}
+function onClickImageFOL(month: string, route: NextRouter) {
+    route.push({ pathname: '/ImageFolder/Detail', query: { month: month } });
+}
+
+function ImageFOL(props: IImageFOL) {
+    const route = useRouter();
+    return (
+        <StyledImageFOL
+            initial="initial"
+            animate="animate"
+            exit="exit"
+            variants={defaultFadeInUpVariants}
+            onClick={() => onClickImageFOL(props.month, route)}
+        >
+            <StyledTestImage />
+            <div>{props.month}</div>
```

Comment:
```suggestion
function ImageFOL({ month, quantity }: IImageFOL) {
    const route = useRouter();
    return (
        <StyledImageFOL
            initial="initial"
            animate="animate"
            exit="exit"
            variants={defaultFadeInUpVariants}
            onClick={() => onClickImageFOL(month, route)}
        >
            <StyledTestImage />
            <div>{month}</div>
```
이렇게 하면 props. 을 하는 수고가 덜어질 것 같은데 이렇게 하신 이유가 혹시 있을까요 ??

Reply:
앗 반영하고 다시 올리겠습니다

Reply:
별 이유는 없었어요